addons/purchase/models/purchase.py

```diff
@@ -335,7 +335,7 @@ def message_post(self, **kwargs):
             self.filtered(lambda o: o.state == 'draft').write({'state': 'sent'})
         po_ctx = {'mail_post_autofollow': self.env.context.get('mail_post_autofollow', True)}
         if self.env.context.get('mark_rfq_as_sent'):
-            po_ctx['mail_notify_author'] = self.env.user.partner_id.id in kwargs.get('partner_ids') or []
+            po_ctx['mail_notify_author'] = self.env.user.partner_id.id in kwargs.get('partner_ids', [])
```

Comment:
Shouldn't it be ``self.env.user.partner_id.id in (kwargs.get('partner_ids') or []`` ? That way a False value in kwargs (may happen) is also replaced by a list. I think it was the purpose, except that some () are missing.